homeworks/serhii.noskov_SerhiiNSN/GamesProject/css/style.css

```diff
@@ -172,3 +172,313 @@ a {
+}
+
+.checkbox-container {
+  display: block;
+  position: relative;
+  padding-left: 35px;
+  color: #FFF;
+  cursor: pointer;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.chekbox-block {
+  display: flex;
+  align-items: center;
+  color: #FFF;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 400;
+  column-gap: 32px;
+}
+
+.checkbox-container input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+.checkmark {
+  position: absolute;
+  top: -2px;
+  left: 0;
+  height: 20px;
+  width: 20px;
+  background-color: #000;
+  border: 1px solid #FFF;
+  border-radius: 6px;
+}
+
+.checkmark::after {
+  content: "";
+  position: absolute;
+  display: none;
+}
+
+.checkbox-container input:checked ~ .checkmark::after {
+  display: block;
+}
+
+.checkbox-container input:checked ~ .checkmark::after {
+  left: 7px;
+  top: 2px;
+  width: 5px;
+  height: 10px;
+  border: solid #FFF;
+  border-width: 0 3px 3px 0;
+  transform: rotate(45deg);
+}
+
+.search-area {
+  position: relative;
+  display: flex;
+}
+
+.search {
+  padding: 14px 14px 14px 46px;
+  outline: none;
+  border: none;
+  border-radius: 6px 0 0 6px;
+  background: #FFF;
+  width: 240px;
+}
+
+.search-img {
+  position: absolute;
+  top: 50%;
+  transform: translate(50%, -50%);
+}
+
+.submit {
+  padding: 13px 20px;
+  border-radius: 0 6px 6px 0;
+  background: #EF4934;
+  border: none;
+  color: #FFF;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 400;
+  cursor: pointer;
+}
+
+/* ========================== LOGIN ============================= */
+
+.login__container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 50px;
+}
+
+.login__tittle {
+  color: #221F1F;
+  font-family: Roboto;
+  font-size: 48px;
+  font-weight: 700;
+  margin: 80px 0 60px;
+  text-align: center;
+}
+
+.email__text,
+.email__password {
+  color: #221F1F;
+  font-family: Roboto;
+  font-size: 22px;
+  font-weight: 600;
+  margin-bottom: 22px;
+}
+
+.email__input,
+.password__input {
+  padding: 30px 380px 30px 20px;
+  border-radius: 6px;
+  border: 1px solid  #A9A9A9;
+  background: #FFF;
+  margin-bottom: 40px;
+  outline: none;
+  font-size: 14px;
+}
+
+.email__input::placeholder {
+  color: #A9A9A9;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.password__input::placeholder {
+  color: #A9A9A9;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.robot__label {
+  color: #221F1F;
+  font-family: Roboto;
+  font-size: 14px;
+  font-weight: 400;
+  margin-left: 12px;
+}
+
+.login__btn {
+  padding: 27px 258px;
+  border-radius: 6px;
+  background: #EF4934;
+  box-shadow: 0 4px 30px 0 rgba(38, 58, 67, .15);
+  border: none;
+  color: #FFF;
+  font-family: Roboto;
+  font-size: 22px;
+  font-weight: 600;
+  cursor: pointer;
+  margin-top: 40px;
+}
+
+.check {
+  display: none;
```

Comment:
I'm afraid, that `display: none` removes the validation tooltip also. So, users could be confused, as to why they can't submit the form
<img width="908" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/a5a879f2-0e98-42e0-a603-44be36106a2c">